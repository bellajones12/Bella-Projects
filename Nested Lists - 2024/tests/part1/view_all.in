NEW 5
hello
1
2
3.14
a
NEW 2
hello world
2e4
VIEW ALL