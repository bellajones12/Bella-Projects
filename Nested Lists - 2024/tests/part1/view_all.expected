List 0: hello -> 1 -> 2 -> 3.14 -> a
List 1: hello world -> 20000.00
Number of lists: 2
List 0
List 1
